### ISSUE TITLE
Support creation of Agents.

### ISSUE THREAD

**Issue by homanp** [homanp/langchain-ui#82]
### Is your proposal related to a problem?

Add possibility for the user to create Agents and connect them to `chatbots` and other tools.

### Describe the solution you'd like

A user should be able to create Agents in a simple ans straightforward way. 

### Describe alternatives you've considered

(Write your answer here.)

### Additional context

The UI focus on connecting different Tools and chains to the agent.

**Comment by finom** [comment]
Unfortunately without agents the UI has no use for me. Clients on the market want to solve 2 groups of tasks:

- Make AI know something (covered by the tool)
- Make AI do something (not covered by the tool or not documented)

The second one isn't less important than the first one. Is there any chance to implement an agent easily? If somebody asks "what's the weather today" I'm going to use location tool and some weather API tool.

**Comment by homanp** [comment]
> Unfortunately without agents the UI has no use for me. Clients on the market want to solve 2 groups of tasks:
> 
> * Make AI know something (covered by the tool)
> * Make AI do something (not covered by the tool or not documented)
> 
> The second one isn't less important than the first one. Is there any chance to implement an agent easily? If somebody asks "what's the weather today" I'm going to use location tool and some weather API tool.

Yes, agree. I created a new repo for agents https://github.com/homanp/superagent

This will also ship with a UI and a Cloud API https://docs.superagent.sh

You can read more here https://superagent.sh
